lib/saml/url_service.rb

```diff
@@ -213,11 +213,12 @@ def build_redirect(params)
     def initialize_tracker(params, previous_saml_uuid: nil)
       previous = previous_saml_uuid && SAMLRequestTracker.find(previous_saml_uuid)
       redirect = previous&.payload_attr(:redirect) || build_redirect(params)
-      inbound = params[:inbound] || nil
+      inbound = previous&.payload_attr(:inbound) || params[:inbound] || nil
```

Comment:
Can `params[:inbound]` be `false`?  Otherwise the `|| nil` is unnecessary.

Reply:
yeah re-reading that I agree, `|| nil` doesn't add anything